ReactWindows/ReactNative/Views/Scroll/ReactScrollViewManager.cs

```diff
@@ -16,6 +17,10 @@ namespace ReactNative.Views.Scroll
     /// </summary>
     public class ReactScrollViewManager : ViewParentManager<ScrollViewer>
     {
+        private bool PagingEnabled = false;
+        private double PageIndex = 0;
+        private Stopwatch stopwatch;
+        
```

Comment:
ReactScrollViewManager manages multiple ScrollViewer instances, so we can't have these properties "global" to all ScrollViewer instances.